src/components/MultiSelect/MultiSelectMenuButton.tsx

```diff
@@ -50,6 +55,8 @@ const MultiSelectMenuButton = forwardRef<
   const iconType = isOpen ? "minus" : "plus";
   const growAnimation = `${grow} 150ms ease-out`;
 
+  const [prevIsOpen, setPrevIsOpen] = React.useState(isOpen);
```

Comment:
I haven't tested this but can you try testing to see if the focus works when `isDefaultOpen` is set to `true` manually? When this is false, the focus check works just fine. I'm curious if when it is open by default, does it still know where to focus when it is closed.

Reply:
Hey @EdwinGuzman. So two things: Manually tested it looks fine. It will have focus on the open MultiSelect (wether it is in a single MultiSelect or a Group) 
Second: should that prop maybe be something that can be changed in the controls in storybook? And in that case maybe isBlockElement too?

<img width="400" alt="Screen Shot 2023-02-07 at 2 38 12 PM" src="https://user-images.githubusercontent.com/70068858/217347817-c1890c07-a6f9-4983-8904-fa51cad30c3c.png">
<img width="260" alt="Screen Shot 2023-02-07 at 2 42 16 PM" src="https://user-images.githubusercontent.com/70068858/217348646-46d4e7d0-94c9-4d7f-a8a0-8917ef189551.png">

Reply:
Great, thanks for checking.

That prop in Storybook doesn't affect the component because they need to be set _before_ the component renders. Otherwise, updating it does nothing. This is hard to test in Storybook and I haven't found a good way to do so with Controls but if you do, you can add it. I usually add an example in the docs; in this case it would be a `MultiSelect` that is already opened on the page when you go to it. That's not necessary for this update and we can create a new ticket for that if you want.